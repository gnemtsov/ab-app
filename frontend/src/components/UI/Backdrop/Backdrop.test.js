import React from 'react';

import { configure, render, mount, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import Backdrop from './Backdrop';

configure({ adapter: new Adapter() });

describe('Backdrop', () => {
	it('Renders correctly when shown', () => {
		const wrapper = shallow(<Backdrop show />);
		expect(wrapper).toMatchSnapshot();
	});
	it('Renders correctly when hidden', () => {
		const wrapper = shallow(<Backdrop />);
		expect(wrapper).toMatchSnapshot();
	});
})