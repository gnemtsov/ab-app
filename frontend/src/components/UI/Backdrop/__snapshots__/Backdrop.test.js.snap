// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Backdrop Renders correctly when hidden 1`] = `""`;

exports[`Backdrop Renders correctly when shown 1`] = `
<div
  className="Backdrop"
/>
`;
